### PR TITLE
release: 4.7.1

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR is to generate a new release of `code-server` at 4.7.1

## Screenshot

image

## TODOs

Follow "Publishing a release" steps in `ci/README.md`

- [ ] publish release and merge PR
- [ ] update the AUR package

### CODE REVIEW

File: ci/helm-chart/Chart.yaml [hunk 1]
```diff
@@ -15,9 +15,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 3.2.2
+version: 3.2.1
```
Comment (code-asher) [inline ci/helm-chart/Chart.yaml:18]: Should this be 3.2.3?  Looks like it got downgraded.

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,18 @@ Code v99.99.999
...
+
+Code v1.71.2
+
+### Changed
+
+- Updated Code to 1.71.2
+
+### Fixed
+
+- Added RPM upgrade flag to install script
```
Comment (code-asher) [inline CHANGELOG.md:33]: We could expand on this a bit:

```suggestion
- Fix the install script not upgrading code-server when already installed on RPM-based machines
```

Comment (code-asher) [reply CHANGELOG.md:33]: Oh also that sudo fix got in with the install script too right?

Comment (code-asher) [reply CHANGELOG.md:33]: `Fix install script installing with su on FreeBSD` or something like that.

Comment (jsjoeio) [reply CHANGELOG.md:33]: Good call! I'll update this now.

File: ci/helm-chart/Chart.yaml (hunk 1 above)
Comment (jsjoeio) [reply ci/helm-chart/Chart.yaml:18]: uhh fog brain, nice catch!

**Review by code-asher — APPROVED** [review]

**Pull request merged**